zabdig

```diff
@@ -638,16 +645,29 @@ class ZInterface(object):
     def get_hosts(self, hostname_needle):
         self._login()
 
-        params = {
-            'output': [
-                'name', 'host', 'interfaces', 'proxy_hostid',
-                'status', 'available'],
-            # 'output': ['name', 'interfaces', 'groups', 'proxy_hostid',
-            #            'status'],
-            'selectInventory': ['host_router'],  # or 'extend' for _all_
-            'selectInterfaces': 'extend',
-            'selectGroups': 'extend',
-        }
+        if (self.version.split(".")[0] >= "7"):
```

Comment:
This string comparison is a bug. It is know to happen that versions increase to above 10 and then suddenly it thinks it's pre-7.

If we had split the version already (as suggested above), this would be:
```
if self.version >= (7,):
```
(Also: no extra parentheses around the entire expression. This is not C/javascript.)